ps5/dropAllConsecutiveEmsgBox.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+   <title>Bitmovin Player Demo</title>
+   <meta charset="UTF-8"/>
+   <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <script type="text/javascript" src="https://cdn.bitmovin.com/player/web/8/bitmovinplayer.js"></script>
+   <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/codem-isoboxer@0.3.7/dist/iso_boxer.min.js"></script>
+   <script type="text/javascript" src="https://cdn.bitmovin.com/player/web/8/modules/bitmovinplayer-playstation5.js"></script>
+   <style>
+     #player {
+       width: 95%;
+       margin: 20px auto;
+       box-shadow: 0 0 30px rgba(0,0,0,0.7);
+     }
+   </style>
+ </head>
+ <body>
+ <div id="player"></div>
+<script type="text/javascript">
+  /**
+   * This example assumes that the EMSG boxes in the segments are consective.
+   * The data sample for EMSG boxes are removed from the segment data and a new
+   * segment data is created.
+   */
+
+  function arrayBufferToBase64(arrayBuffer) {
+    return btoa(String.fromCodePoint(...new Uint8Array(arrayBuffer)));
+  }
+
+  function base64ToArrayBuffer(base64) {
+    const binaryString = window.atob(base64);
+    const length = binaryString.length;
+    const bytes = new Uint8Array(length);
+    for (let i = 0; i < length; i++) {
+      bytes[i] = binaryString.charCodeAt(i);
+    }
+
+    return bytes.buffer;
+  }
+
+  function onMetadataEvent (event) {
+    console.log('onMetadata : type=' + event.metadataType);
+  }
+
+  var bufferConfig = {};
+  var bufferLevels = {};
+
+  bufferLevels['forwardduration'] = 12;
+  bufferLevels['backwardduration'] = 12,
+
+  bufferConfig['video'] = bufferLevels;
+  bufferConfig['audio'] = bufferLevels;
+
+  var conf = { 
+    key: '6c88f166-fe3e-467a-9276-aea43864cdcb',
```

Comment:
Is this key allowed to be public?

Reply:
I think not. I am working on squashing the commits and force pushing to rewrite the history.

Reply:
Although I think we sometime share the code with this key to our customer.